tests/scripts/components-configuration-tls.sh

```diff
@@ -683,4 +683,9 @@ component_test_depends_py_kex () {
     tests/scripts/depends.py kex
 }
 
-
+component_test_config_minimal_tls1_3 () {
+    msg "build: configs/config-aes-gcm-tls1_3.h"
+    MBEDTLS_CONFIG="configs/config-aes-gcm-tls1_3.h"
+    CC=$ASAN_CC cmake -D GEN_FILES=Off -DMBEDTLS_CONFIG_FILE="$MBEDTLS_CONFIG" -D CMAKE_BUILD_TYPE:String=Asan .
+    make
+}
```

Comment:
Please run `make test`. It's expected to pass in all configurations. It doesn't have a lot of TLS testing, but it has some, and that's better than nothing.